manual_tests/lib/gh/check.py

```diff
@@ -41,6 +42,7 @@ def get_runs_json(pr_url: URL) -> Generator[json.Value]:
 @dataclass(frozen=True)
 class Check:
     pr: PR
+    workflow_name: WorkflowName
```

Comment:
```suggestion
    workflow: WorkflowName
```

Unless there's some ambiguity, i try to avoid "_name" in names.

(Obviously, with the one exception that I find `Check.name: CheckName` much more intuitive than `Check.check: CheckName`.)

Reply:
I called it workflow name to match what it is called in the json since workflow sounds like it could be an object of its own

Reply:
I named it so that it would match the property in check run. Additionally, I felt that `workflow` could be an object of its own and `workflow_name` is more accurate in my opinion.